lib/kafka/broker_pool.rb

```diff
@@ -2,12 +2,14 @@
 
 module Kafka
   class BrokerPool
-    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:)
+    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:, ssl: nil, ssl_context: nil)
```

Comment:
Awesome! I'll just edit stuff after merging if it needs to be consistent with the writing style of the README.